docs/csp.md

```diff
@@ -41,45 +50,55 @@ app.use(
 When loading the initial index page, the `Content-Security-Policy` header should now contain the nonce:
 
 ```
-style-src 'self' 'nonce-N2M0MDhkN2EtMmRkYi00MTExLWFhM2YtNDhkNTc4NGJhMjA3'
+default-src 'self'; style-src 'self' 'nonce-N2M0MDhkN2EtMmRkYi00MTExLWFhM2YtNDhkNTc4NGJhMjA3';
 ```
 
-2. In webpack config:
+1. Now we want to include this value in our page so that JSS can pick it up at runtime.
+
+You can use any templating engine, or if you have SSR that should work too. For this example we'll use the [Nunjucks](https://github.com/mozilla/nunjucks) template engine.
 
-Set the `global.__webpack_nonce__` variable to the same nonce value:
+1. First create the template html file.
+
+```html
+<head>
+  <meta id="jss-nonce" data-value="{{ noncePlaceholder }}">
+</head>
+...
+```
+
+1. Now update the server to render the template with the `noncePlaceholder` variable to be interpolated with the nonce generated from our middleware `res.locals.nonce`.
 
 ```js
-// webpack.config.js
-const WEBPACK_NONCE = 'N2M0MDhkN2EtMmRkYi00MTExLWFhM2YtNDhkNTc4NGJhMjA3'
-module.exports = {
-  /* ... */
-  plugins: {
-    new webpack.DefinePlugin({
-      'global.__webpack_nonce__': JSON.stringify(WEBPACK_NONCE),
-    }),
-  },
-}
+const express = require("express");
+const expressNunjucks = require("express-nunjucks");
+//...
+app.get("/", function(req, res) {
+  res.render("index", { noncePlaceholder: res.locals.nonce });
+});
 ```
 
-(`__webpack_nonce__` is a poorly documented variable which, when set, JSS will read and apply to `<style />` elements: `<style nonce={webpack-nonce-value} />`).
+The tag must have `id="jss-nonce"` and have a `data-value` attribute with the string value to use as the nonce.
+
+Now JSS can apply the nonce to `<style />` elements: `<style nonce={webpack-nonce-value} />`
 
 At this point, the style blocks generated by JSS should render with the nonce value (`<style nonce />`) when you inspect the page.
 
 The browser might not show the nonce value inside the style tag when you inspect the page, but it's there regardless.
 
 ## Setting the nonce attribute in styling loaded by Webpack
```

Comment:
Is this still appropriate?

Reply:
I'm going to assume you meant the webpack reference, I removed it now